.github/workflows/pr-open-workflow.yml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+name: PR Open Workflow
+run-name: ${{ github.actor }} pull request
+on:
+  pull_request:
+    types:
+      - opened

```

Comment:
`ready_for_review` should also be included, perhaps add something to the message indicating the type of PR. I'm not sure, but it may want them listed as `types: [opened, ready_for_review]`.